index.js

```diff
@@ -139,7 +139,7 @@ prog
 
     const errArray = []
     const globOpts = { nodir: true }
-    const acornOpts = { ecmaVersion, silent: true }
+    const acornOpts = { ecmaVersion: parseInt(ecmaVersion, 10), silent: true }
```

Comment:
I suspect this is why tests were failing. Acorn does a number of strict equality checks where it assumes ecmaVersion is a number, not a string. Without this, it defaults to es3, and so the es5 test fails due to the reserved es3 keyword "char".